style/components/post.css

```diff
@@ -0,0 +1,74 @@
+.main__post {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 2fr 1fr;
+
+  color: var(--post-color);
+
+  /* font-weight: 400;
```

Comment:
on screen in dev tools the font size seems to be to bold (iPhone SE), but on bigger phones it looks perfectly. You could experiment with font-weight: 300 (or 200)